src/core/server/saved_objects/migrations/actions/index.ts

```diff
@@ -35,11 +35,12 @@ export { removeWriteBlock } from './remove_write_block';
 export type { CloneIndexResponse, CloneIndexParams } from './clone_index';
 export { cloneIndex } from './clone_index';
 
-export type {
-  WaitForIndexStatusYellowParams,
+export type { WaitForIndexStatusParams, IndexNotYellowTimeout } from './wait_for_index_status';
+import {
+  IndexNotGreenTimeout,
```

Comment:
NIT `IndexNotGreenTimeout` (and `IndexNotYellowTimeout`) should be imported as a type.